presto-docs/src/main/sphinx/release/release-0.181.rst

```diff
@@ -0,0 +1,68 @@
+=============
+Release 0.181
+=============
+
+General Changes
+---------------
+
+* Fix query failure and untracked memory usage when query contains
+  :func:`transform_keys` or :func:`transform_values`.
+* Fix a bug that may cause ``CREATE TABLE IF NOT EXISTS`` queries to fail
+  with *"Table already exists"*.
+* Fix query failure when ``ORDER BY`` expressions reference columns that are used in
+  the ``GROUP BY`` clause by their fully-qualified name.
+* Improve error handling when passing too many arguments to various
+  functions or operators that take a variable number of arguments.
+* Improve performance of ``count(*)`` aggregations over subqueries with known
+  constant cardinality.
+* Add support for ``VERBOSE`` modifier to ``EXPLAIN ANALYZE`` command for more detailed,
```

Comment:
This phrasing is off. How about
```
* Add ``VERBOSE`` option for :doc:`/sql/explain-analyze` that provides additional low-level details about query performance.
```
Also, this new option isn't documented. We need to merge that before the release. I remember reviewing it, let me find it.